Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3
+
+WORKDIR /procrastinate_dev/
+COPY . ./
```

Comment:
Go back to copying only requirements/setup/procrastinate, and remove dockerignore.